fbw-a32nx/src/systems/fmgc/src/flightplanning/FlightPlanManager.ts

```diff
@@ -785,6 +787,7 @@ export class FlightPlanManager {
         }
 
         this.updateFlightPlanVersion().catch(console.error);
+        NXDataStore.set('PLAN_DESTINATION', ICAO.getIdent(icao));
```

Comment:
Hmmm, this should not be needed. NXDataStore is for persistent storage of settings etc.

Reply:
Discussed with holland. This was an old hack before there was a better way to do cross-instrument sync so we can keep for now.